### PR TITLE
[MIG] portal_multicurrency_totals: Migration to 17.0 t#79454

### PR DESCRIPTION
**Pull request by CLaurelB**

### CODE REVIEW

**Review by desdelinux — APPROVED** [review]

**Comment by desdelinux** [comment]
Hi @hugho-ad Please review.

**Review by hugho-ad — APPROVED** [review]

File: portal_multicurrency_totals/README.rst [hunk 1]
```diff
@@ -16,13 +16,13 @@ To use this module, just go to the portal  and open a document list; you will se
 When accessing quotations or sale orders, a total will be displayed per
 currency:
 
-    .. image:: portal_multicurrency_totals/static/description/totals_in_so.png
+    .. image:: /portal_multicurrency_totals/static/description/totals_in_so.png
```
Comment (luisg123v) [inline portal_multicurrency_totals/README.rst:19]: Why this change?

Please check if this image is being rendered correctly on GitHub.

File: portal_multicurrency_totals/demo/res_partner_demo.xml [hunk 2]
```diff
@@ -2,24 +2,21 @@
...
 
     <!-- Grant portal access to the customer -->
-    <record id="grant_portal_access" model="portal.wizard">
+    <record id="grant_portal_access" model="portal.wizard.user">
         <field
-            name="user_ids"
+            name="user_id"
             model="res.partner"
             eval="[
             (0, 0, {
```
Comment (luisg123v) [inline portal_multicurrency_totals/demo/res_partner_demo.xml]: Migrate to the command syntax.

And let's take the chance to fix indentation here.

File: portal_multicurrency_totals/demo/res_partner_demo.xml [hunk 3]
```diff
@@ -35,10 +32,6 @@
         ]"
         />
     </record>
-    <function
-        model="change.password.wizard"
-        name="change_password_button"
-        eval="[ref('set_password_portal_user')]"
-    />
+    <function model="change.password.wizard" name="change_password_button" eval="[ref('set_password_portal_user')]" />
```
Comment (luisg123v) [inline portal_multicurrency_totals/demo/res_partner_demo.xml:38]: Perform autoformatting changes on a separate commit located before this one, as we would do when migratin an OCA module.

**Review by luisg123v — CHANGES_REQUESTED** [review]

File: portal_multicurrency_totals/README.rst (hunk 1 above)
Comment (CLaurelB) [reply portal_multicurrency_totals/README.rst:19]: I changed it because the image was being shown as a broken link in the previous version.

File: portal_multicurrency_totals/demo/res_partner_demo.xml (hunk 2 above)
Comment (CLaurelB) [reply portal_multicurrency_totals/demo/res_partner_demo.xml]: Fixed.

File: portal_multicurrency_totals/demo/res_partner_demo.xml (hunk 3 above)
Comment (CLaurelB) [reply portal_multicurrency_totals/demo/res_partner_demo.xml:38]: Done.

Comment (luisg123v) [reply portal_multicurrency_totals/demo/res_partner_demo.xml:38]: Please include in the history all commits merged in v14 (#1648).

Comment (CLaurelB) [reply portal_multicurrency_totals/demo/res_partner_demo.xml:38]: Done.

**Review by luisg123v — APPROVED** [review]
LGTM, but this is red.

Could you fix it in a separate PR, please?

Just cherry-pick c84e8fe24bffb47eeae5bbc9fa0c4bd111397d0b

**Comment by CLaurelB** [comment]
> LGTM, but this is red.
> 
> Could you fix it in a separate PR, please?
> 
> Just cherry-pick [c84e8fe](https://github.com/Vauxoo/addons-vauxoo/commit/c84e8fe24bffb47eeae5bbc9fa0c4bd111397d0b)

Done in https://github.com/Vauxoo/addons-vauxoo/pull/1650

**Comment by luisg123v** [comment]
@CLaurelB thanks, just rebase.

**Comment by CLaurelB** [comment]
> @CLaurelB thanks, just rebase.

Done.
